doc/explanation/api/examples/outliers_declarative.md

```diff
@@ -82,3 +82,40 @@ pn.Column(obj.param, obj.view)
 ```
 
 To support various domains, you can create hierarchies of classes encapsulating parameters and functionality across different object families. Parameters and code can inherit across classes as needed, without depending on any specific GUI library. This approach facilitates the maintenance of large codebases, all displayable and editable with Panel, adaptable over time. For a more complex illustration, refer to the [Attractors Panel app](https://examples.holoviz.org/gallery/attractors/attractors_panel.html) ([source](https://github.com/holoviz-topics/examples/tree/main/attractors)), and explore the Panel codebase itself for extensive usage of Param throughout the codebase.
+
+## Serving the Notebook
+
+Lets finalize our app by organizing our components in a nicely styled template (`MaterialTemplate`) and mark it `.servable()` to add it to our served app:
+
+```python
+pn.template.MaterialTemplate(
+    site="Panel",
+    title="Getting Started App",
+    sidebar=[obj.param],
+    main=[obj.view],
+).servable(); # The ; is needed in the notebook to not display the template. Its not needed in a script
+```
+
+Save the notebook with the name `app.ipynb`.
+
+Finally, we'll serve the app by running the command below in a terminal:
+
+```bash
+panel serve app.ipynb --autoreload
```

Comment:
Unfortunately the getting started app displays 

```bash
2024-07-30 06:41:20,719 Dropping a patch because it contains a previously known reference (id='p1269'). Most of the time this is harmless and usually a result of updating a model on one side of a communications channel while it was being removed on the other end.
2024-07-30 06:41:20,956 Dropping a patch because it contains a previously known reference (id='p1369'). Most of the time this is harmless and usually a result of updating a model on one side of a communications channel while it was being removed on the other end.
2024-07-30 06:41:21,789 Dropping a patch because it contains a previously known reference (id='p1551'). Most of the time this is harmless and usually a result of updating a model on one side of a communications channel while it was being removed on the other end.
2024-07-30 06:41:22,023 Dropping a patch because it contains a previously known reference (id='p1646'). Most of the time this is harmless and usually a result of updating a model on one side of a communications channel while it was being removed on the other end.
2024-07-30 06:41:22,255 Dropping a patch because it contains a previously known reference (id='p1738'). Most of the time this is harmless and usually a result of updating a model on one side of a communications channel while it was being removed on the other end.
2024-07-30 06:41:22,501 Dropping a patch because it contains a previously known reference (id='p1828'). Most of the time this is harmless and usually a result of updating a model on one side of a communications channel while it was being removed on the other end.
2024-07-30 06:41:22,740 Dropping a patch because it contains a previously known reference (id='p1918'). Most of the time this is harmless and usually a result of updating a model on one side of a communications channel while it was being removed on the other end.
2024-07-30 06:41:23,807 Dropping a patch because it contains a previously known reference (id='p2106'). Most of the time this is harmless and usually a result of updating a model on one side of a communications channel while it was being removed on the other end.
2024-07-30 06:41:24,047 Dropping a patch because it contains a previously known reference (id='p2207'). Most of the time this is harmless and usually a result of updating a model on one side of a communications channel while it was being removed on the other end.
```

This is highly confusing for a new user. Even though it states it probably not a problem its overload of information and signals that something is not fully robust.

Reply:
Can't reproduce this.

Reply:
I'm on the `main` branch of Panel and get this when I drag the top left slider. hvplot==0.10.0, holoviews==1.19.1,bokeh=3.5.0.

https://github.com/user-attachments/assets/da3369db-360a-4363-af48-8a5ac8db6dbb

```python
import hvplot.pandas
import numpy as np
import pandas as pd
import panel as pn

PRIMARY_COLOR = "#0072B5"
SECONDARY_COLOR = "#B54300"
CSV_FILE = (
    "https://raw.githubusercontent.com/holoviz/panel/main/examples/assets/occupancy.csv"
)

pn.extension(design="material", sizing_mode="stretch_width")

@pn.cache
def get_data():
  return pd.read_csv(CSV_FILE, parse_dates=["date"], index_col="date")

data = get_data()

def transform_data(variable, window, sigma):
    """Calculates the rolling average and identifies outliers"""
    avg = data[variable].rolling(window=window).mean()
    residual = data[variable] - avg
    std = residual.rolling(window=window).std()
    outliers = np.abs(residual) > std * sigma
    return avg, avg[outliers]


def get_plot(variable="Temperature", window=30, sigma=10):
    """Plots the rolling average and the outliers"""
    avg, highlight = transform_data(variable, window, sigma)
    return avg.hvplot(
        height=300, legend=False, color=PRIMARY_COLOR
    ) * highlight.hvplot.scatter(color=SECONDARY_COLOR, padding=0.1, legend=False)

variable_widget = pn.widgets.Select(name="variable", value="Temperature", options=list(data.columns))
window_widget = pn.widgets.IntSlider(name="window", value=30, start=1, end=60)
sigma_widget = pn.widgets.IntSlider(name="sigma", value=10, start=0, end=20)

bound_plot = pn.bind(
    get_plot, variable=variable_widget, window=window_widget, sigma=sigma_widget
)

pn.template.MaterialTemplate(
    site="Panel",
    title="Getting Started App",
    sidebar=[variable_widget, window_widget, sigma_widget],
    main=[bound_plot],
).servable(); # The ; is needed in the notebook to not display the template. Its not needed in a script
```